modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/method_init_composed.mustache

```diff
@@ -17,6 +17,11 @@
         self._path_to_item = _path_to_item
         self._configuration = _configuration
 
+        if self._configuration.discard_unknown_keys:
```

Comment:
Can you handle the case when _configuration is none?

Reply:
done.